### DISCUSSION TITLE
Only c5cb58328c555ac27679422b1da940a9b19de6f2 commit allows to perform Lora training on 2060RTX 6GB

### DISCUSSION THREAD

**Discussion by healthyfat** [d8ahazard/sd_dreambooth_extension#856] [Q&A]
None of the updates I tried works. I had to go back to the commit from December. 

Anyone knows newer commit that works on 6GB VRAM?

**Comment by Zuxier** [comment]
newer commits train properly, results on previous commits is not guaranteed

**Reply by healthyfat** [reply]
I can't get any newer commit then above to perform Lora training on Nvidia 2060 6GB card.

**Comment by d8ahazard** [comment]
You saying you didn't even try the latest version?

**Comment by healthyfat** [comment]
No. I did try latest version. Actually few of them in last few days. Still had to go back to December commit to be able to perform any Lora training.
